tests/cypress/e2e/unit_tests/p0_fleet.spec.ts

```diff
@@ -164,7 +206,100 @@ describe('Test Fleet deployment on PRIVATE repos using KNOWN HOSTS', { tags: '@p
         cy.contains('Ssh: handshake failed: knownhosts: key mismatch').should('be.visible');
     })
   );
-});
+
+  // Default verify
+  qase(168,
+    it('FLEET-168 Verify Fleet default known-host is set on configmap',
+      { tags: '@fleet-168' }, () => {
+
+        // Create private repo using known host
+        cy.accesMenuSelection('local', 'Storage', 'ConfigMaps');
+        cy.nameSpaceMenuToggle('All Namespaces');
+        cy.filterInSearchBox('known-hosts');
+        cy.open3dotsMenu('known-hosts', 'Edit YAML');
+        cy.contains('ssh-rsa').its(length).should('be.visible')
+    })
+  );
+
+  // No custom / yes default
+  qase(169,
+    it('FLEET-169 Verify that without custom known-host, fleet uses default custom ones from defined in configmap',
+      { tags: '@fleet-169' }, () => {
+
+        const repoName = 'local-cluster-fleet-169';
+        const gitAuthType = "ssh"
+        const repoUrl = "git@github.com:fleetqa/fleet-qa-examples.git"
+        const userOrPublicKey = Cypress.env("rsa_public_key_qa")
+        const pwdOrPrivateKey = Cypress.env("rsa_private_key_qa")
```

Comment:
Can be placed at describe level and can reduce its duplication?

Reply:
Sure thing. Will do in separate pr